plugins/gossip/autopeering.go

```diff
@@ -46,27 +48,31 @@ func configureAutopeering() {
 		if !ev.Status {
 			return // ignore rejected peering
 		}
-		gossipAddr := ev.Peer.Services().Get(services.GossipServiceKey()).String()
-		apLog.Infof("[incoming peering] whitelisting %s / %s / %s", ev.Peer.Address(), gossipAddr, ev.Peer.ID())
+		gossipAddr := ev.Peer.Services().Get(services.GossipServiceKey())
```

Comment:
technically, this is no longer an "address" but only network (tcp or udp) plus a port. So consider renaming to gossipService or something similar